src/index.js

```diff
@@ -88,7 +88,7 @@ export default {
     recommended,
   },
   rules: _.mapValues(rules, (rule, key) => {
-    if (key === 'no-types-missing-file-annotation') {
+    if (['no-types-missing-file-annotation', 'require-valid-file-annotation'].includes(key)) {
```

Comment:
Had to change the way how `require-valid-file-annotation` works with `onlyFilesWithFlowAnnotation` setting, because of breaking change in `checkFlowFileAnnotation.js`